app/controllers/idv/phone_controller.rb

```diff
@@ -55,6 +55,7 @@ def max_attempts_reached
         throttle_type: :proof_address,
         step_name: step_name,
       )
+      irs_attempts_api_tracker.idv_phone_otp_sent_rate_limited
```

Comment:
Adding this here because the analytics_tracker is logging a similar event here.

Reply:
Just to ask the obligatory question -- this doesn't lead to a duplicate, does it? These are two separate cases where we need to log this?

Reply:
This does not lead to duplicate event logging.  It seems like a separate throttle path that I thought should be captured...

Reply:
I'm just going to remove this, since it is not needed.